Makefile

```diff
@@ -52,12 +60,16 @@ fmt:
 vet:
 	go vet ./...
 
+# Run golangci-lint agaist code
+lint: vet
+	# golangci-lint run --fix
+
 # Generate code
 generate: controller-gen
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
 # Build the docker image
-docker-build: test
+docker-build:
```

Comment:
Do you want to build even if the test are not passing?

Reply:
I don't want to duplicate `test` stage when building the docker image in CI.